packetbeat/_meta/beat.full.yml

```diff
@@ -432,23 +438,22 @@ packetbeat.protocols.nfs:
 #
 # Process matching is optional and can be enabled by uncommenting the following
 # lines.
-#
-#packetbeat.procs:
-#  enabled: false
-#  monitored:
-#    - process: mysqld
-#      cmdline_grep: mysqld
-#
-#    - process: pgsql
-#      cmdline_grep: postgres
-#
-#    - process: nginx
-#      cmdline_grep: nginx
-#
-#    - process: app
-#      cmdline_grep: gunicorn
+packetbeat.procs:
+  enabled: false
+  monitored:
+    - process: mysqld
+      cmdline_grep: mysqld
+
+    - process: pgsql
+      cmdline_grep: postgres
+
+    - process: nginx
+      cmdline_grep: nginx
+
+    - process: app
+      cmdline_grep: gunicorn
 
-# Uncomment the following if you want to ignore transactions created
+# Set the following to true if you want to ignore transactions created
```

Comment:
Ah, yes, that should have been under "General", I probably copied it there by mistake. There is already a "General" section, btw.